src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1754,5 +1754,7 @@
   "com.minecolonies.coremod.gui.townhall.edit_flag": "Edit colony flag",
   "block.minecolonies.colony_banner": "Colony Flag Banner",
   "com.minecolonies.coremod.gui.locate": "Locate",
-  "com.minecolonies.coremod.locating": "Successfully located rack with respective Stack..."
+  "com.minecolonies.coremod.locating": "Successfully located rack with respective Stack...",
+
+  "entity.researcher.moreknowledge": "During your absence your researchers used the time to do some extra research which will speed up your current research a bit!"
```

Comment:
```suggestion
  "entity.researcher.moreknowledge": "During your absence, your Researchers used the time to do some extra work. This will speed up their current research a bit!"
```